package.json

```diff
@@ -24,23 +24,22 @@
   },
   "scripts": {
     "postinstall": "cd src/svelte && yarn install",
-    "gen-version-ts": "run-func build/scripts/yarn-scripts.ts genVersionTS",
-    "nodeclean": "run-func build/scripts/yarn-scripts.ts nodeclean",
-    "scalaclean": "run-func build/scripts/yarn-scripts.ts scalaclean",
+    "gen-version-ts": "run-func build/yarn-scripts.ts genVersionTS",
+    "nodeclean": "run-func build/yarn-scripts.ts nodeclean",
+    "scalaclean": "run-func build/yarn-scripts.ts scalaclean",
     "clean": "yarn nodeclean && yarn scalaclean",
     "precompile": "yarn nodeclean && yarn gen-version-ts && cd src/svelte && yarn build",
     "compile": "tsc -p ./ && yarn sbt",
     "lint": "yarn prettier src -c && cd src/svelte && yarn lint",
     "lint:fix": "yarn prettier src -w && cd src/svelte && yarn lint:fix",
     "scalawatch": "nodemon --config debugger/nodemon.json",
-    "prewatch": "yarn gen-version-ts",
-    "watch": "run-func build/scripts/yarn-scripts.ts watch",
+    "prewatch": "yarn gen-version-ts && yarn sbt",
+    "watch": "run-func build/yarn-scripts.ts watch",
     "watch:svelte": "cd src/svelte && yarn dev",
-    "webpack": "webpack --mode production --config ./build/extension.webpack.config.js",
-    "prepackage": "yarn install && yarn compile && yarn webpack",
-    "package": "yarn package-setup && yarn package-create",
-    "package-setup": "run-func build/scripts/package.ts setup",
-    "package-create": "run-func build/scripts/package.ts create",
+    "webpack": "webpack --mode production --config ./webpack/ext-dev.webpack.config.js",
+    "webpack:pkg": "webpack --mode production --config ./webpack/ext-package.webpack.config.js",
+    "prepackage": "yarn install && yarn compile && yarn webpack:pkg",
+    "package": "run-func build/yarn-scripts.ts package",
```

Comment:
Put the `yarn` calls at the end of that `package` function here in this line instead of calling back to `yarn` and creating a cycle.

Reply:
As mentioned https://github.com/apache/daffodil-vscode/pull/708#discussion_r1270653896 this may need to be discussed before changing it. Or are you only saying to change this for the package yarn commands and all the others are fine?

Reply:
Just saying that fix the `package` `yarn` script to have **it** call the other `yarn` `commands` instead of calling the `yarn` commands inside of the `yarn-scripts.ts` file

Reply:
I think we can then get rid of the `execSync` dependency once this change is done too.